src/cdn/lite.js

```diff
@@ -0,0 +1,13 @@
+import { Agent } from "../loaders/agent";
```

Comment:
lite, pro, and spa loaders use `agent` loader.  This is the "custom" loader that you can feed features into.